src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -227,9 +227,9 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
             }
         }
 
-    override fun getPersonerWithOppgaveAndOldEnhet(): List<Pair<PersonIdent, String?>> =
+    override fun getPersonerToUpdateEnhet(): List<Pair<PersonIdent, String?>> =
```

Comment:
Bør denne hete hva den gjør eller hva vi bruker funsjonen til?🤔 Synes førstnevnte høres fornuftig ut. Det den gjør er å hente personer som er aktive uten tildelt enhet?

Reply:
Er dette personer man vet må oppdateres eller innhenter man enhet på disse personer basert på kriterier beskrevet i PR? For lurte litt på om man kunne brukt `utdatert` i denne konteksten.

Reply:
Outdated/utdatert hadde nok passet bedre ja :+1: